playground/hmr/circular/mod-b.js

```diff
@@ -0,0 +1,3 @@
+import { value as _value } from './mod-c'
+
+export const value = `mod-b -> ${_value}`
```

Comment:
If I add `import.meta.hot?.accept(() => {})` here, the full refresh starts happening.
```
index.js (boundary) -> mod-a.js -> mod-b.js (boundary) -> mod-c.js -> mod-a.js
```
It might be good to improve this case as well in future.

Reply:
Yeah this one is tricky currently since the two boundary could start fetching at the same time, causing potential execution order issues. We _could_ detect and only refresh `index.js` as the boundary, but I think it's also nice that this triggers a page reload + HMR debug logs so they can fix this and get fine-grained HMR in the first place.

Reply:
> I think it's also nice that this triggers a page reload + HMR debug logs so they can fix this and get fine-grained HMR in the first place.

Ah, that's a good point.